src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,31 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao =
+                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        Manufacturer manufacturerForCreate = new Manufacturer();
+        manufacturerForCreate.setName("Bob");
+        manufacturerForCreate.setCountry("USA");
+        Manufacturer savedFormat = manufacturerDao.create(manufacturerForCreate);
+        System.out.println(savedFormat);
+
+        Manufacturer manufacturerForUpdate = new Manufacturer();
+        manufacturerForUpdate.setId(5L);
```

Comment:
Давай використовувати `Id`  вже збереженого Manufacturer, який точно є в базі
```suggestion
        manufacturerForUpdate.setId(savedManufacturer.getId());
```